nrt/data/SVarC/description/SVarC.tex

```diff
@@ -155,8 +155,8 @@ \subsection{Step on the reference voltage}
   \begin{tikzpicture}
     \begin{axis}[height = 1.7in]
        \addplot[color=blue!50]
-       table[x=time,y expr=\thisrow{Test_svarc_injector_UPu}*225]
-       {../SVarC_1_StepUref/reference/outputs/curves/curves.csv};
+       table[x=time,y expr=\thisrow{SVarC_SVarC_injector_UPu}*225]
```

Comment:
Why this change ?

Reply:
The .tex could not be compiled, the variables had not been updated after a previous modification.